plugins/synapticsmst/synapticsmst.quirk

```diff
@@ -1,3 +1,27 @@
+[SynapticsMSTBoardID=0]
+Name = Synaptics EVB development board
+GuidTemplate = evb
```

Comment:
I'm not 100% sold on the "GuidTemplate" name -- got any other ideas?

Reply:
`DeviceType` ?

Reply:
`DockType`, `DockKind` or `DeviceKind` might be safer; device_type is used for the g_type stuff.

Reply:
I think `DeviceKind` keeps it agnostic in case these things are in dongles or some other thing we can't fathom yet ever.

Reply:
Works for me.